openvpn/middlewares/server/filter/middleware_test.go

```diff
@@ -21,18 +21,19 @@ import (
 	"testing"
 
 	"github.com/mysteriumnetwork/go-openvpn/openvpn/management"
+	"github.com/mysteriumnetwork/go-openvpn/openvpn/middlewares/server"
 	"github.com/stretchr/testify/assert"
 )
 
 const (
-	emptyFilter = `client-pf 0
+	emptyFilter = `client-pf 3
```

Comment:
What is the point to change it from 0 to 3?

Reply:
Was triggering different events in tests. Reverted